src/Common/src/System/Net/NetworkInformation/UnixCommandLinePing.cs

```diff
@@ -4,6 +4,7 @@
 
 using System.IO;
 using System.Text;
+using System.Globalization;
```

Comment:
nit: Move this to the top of the list of using directives. We sort these alphabetically, usually.